README.rst

```diff
@@ -43,6 +43,49 @@ __ `OpenBSD package`_
 __ `openSUSE package`_
 __ `Ubuntu package`_
 
+Docker
+------
+
+A docker package exists for this project::
+
+  $ git pull ghcr.io/rss2email/rss2email:master
+
+Then it can be used like::
+
+  $ docker run  -v ./data:/data -v ./config:/config rss2email list
+  
+You need to create the config file and have some kind of SMTP server set up in the config file. With ``--net host`` option you can use SMTP server on the host.
+
+Docker compose
+~~~~~~~~~~~~~~
+
+Here is an example docker-compose.yaml::
+
+  version: '2'
+  services:
+    rss2email:
+      image: ghcr.io/rss2email/rss2email:master
+      volumes:
+        - ./config:/config
+        - ./data:/data
+      entrypoint: "/venv/bin/python3 -c 'import threading; threading.Event().wait()'"
```

Comment:
@plantroon 
A later paragraph mentions there is no shell in the container, but this looks like a shell command line to me. Educate me here — does docker-compose parse this line and call `exec` with the parsed parameters?

@auouymous 
To answer your opening question, no, I do not have much hands-on experience with Docker.

Reply:
@amiryal yes, this is executed directly by Docker (no need for compose, specifying entrypoint in docker run would yield the same result) even without a shell available in the container.

The rationale behind this is for it to keep running as a service so that a job scheduler can pick it up and exec `r2e run`, which to my understanding runs the same way as when you would call `docker exec`.

Reply:
@amiryal  I understand the confusion now. I specified it in the shell command form, therefore it runs through a shell - and even distroless images have `sh` binary in them by default, but I thought they didn't.

I will change this to use the exec command form to make it cleaner.